pkgs/by-name/ni/nixops-dns/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/kamilchm/nixops-dns
```

Comment:
Shouldn't these files be upstreamed somehow?

Reply:
Upstream kinda unmaintained.

Reply:
Yes, I just clicked archive on the nixops-dns repo because I haven't used it in years. The best thing to do would be to fork it and add these files there. It's a pretty simple tool, and it's less than 100 lines of Go. It should be easy to modify and maintain for anyone who needs it. Thx @aaronjheng  for taking care of it in the nixpkgs.

Reply:
Well, initially I would suggest a final, definitive 1.1 for this.
But your idea of fork is way better! I will [museoa](https://github.com/museoa) it!
(I would prefer nix-community but I don't know how to do it)

Reply:
Untested:

https://github.com/museoa/nixops-dns/releases/tag/v1.0.1